conda/repodata.py

```diff
@@ -0,0 +1,99 @@
+# (c) Continuum Analytics, Inc. / http://continuum.io
+# All Rights Reserved
+#
+# conda is distributed under the terms of the BSD 3-clause license.
+# Consult LICENSE.txt or http://opensource.org/licenses/BSD-3-Clause.
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+from .core import repodata as _impl
+from .connection import CondaSession
+from os.path import join
+
+
+class RepoData(object):
```

Comment:
I was just in general commenting on the convention that a collection of like objects ends in 's'.  Which is actually why the return value of the current `collect_all_repodata()` function is named `repodatas`, and then that return value is iterated over in the current `fetch_index()` function.